.github/workflows/tests.yml

```diff
@@ -40,8 +40,6 @@ jobs:
                 max_attempts: 3
                 command: composer update ${{ matrix.composerflags }}
             - name: Run script
-              env:
-                GOOGLE_APPLICATION_CREDENTIALS: ${{github.workspace}}/tests/Tests/Unit/testdata/json-key-file.json
```

Comment:
same, without checking workflow it's pretty much impossible to figure out what's wrong so I just moved it to phpunit.xml.dist